clients/extension/popup.html

```diff
@@ -10,59 +10,55 @@
   <meta charset="utf-8">
   <meta name="author" content="Lighthouse Team">
   <meta name="viewport" content="width=device-width">
-  <title>Lighthouse Report</title>
+  <title>Lighthouse</title>
   <link rel="stylesheet" href="styles/lighthouse.css">
-  <link rel="stylesheet" href="styles/lighthouse-loading.css">
 </head>
 <body>
-
-  <header class="header">
-    <img src="images/lh_logo_bg.png" class="header__icon" width="100" height="100" alt="Lighthouse Logo">
-    <div class="header-titles">
-      <h1 class="header-titles__main">Lighthouse</h1>
-      <h2 class="header-titles__url">...</h2>
-      <div class="header-titles__throttling">
-        <label for="lantern-checkbox">
-          <input
-            id="lantern-checkbox"
-            type="checkbox"
-            checked />
-          Simulated throttling (faster)
-        </label>
-      </div>
-    </div>
-  </header>
-
   <main class="main" role="main">
-    <div class="feedback"></div>
-
-    <button class="button button--configure" id="configure-options">Options</button>
-    <button class="button button--generate" id="generate-report">Generate report</button>
-  </main>
-
-  <!-- Show running view initially (.subpage--visible). -->
-  <aside class="status subpage subpage--visible">
-    <div class="lighthouse-effects">
-      <img src="images/lh_logo_bg_no-light.png" alt="LH logo">
-      <div class="lighthouse-effects__wrapper">
-        <div class="lighthouse-effects__light"></div>
+    <div class="section section--run">
+      <img src="images/lh_logo_128x128.png" class="header__icon" width="75" height="75" alt="Lighthouse Logo">
```

Comment:
this 128px is 5k bigger than the SVG logo. :p 

(it also looks a lil blurry to me)